src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ name }) => (
+  React.createElement('button', { type: 'button' }, name)
+);
```

Comment:
__[Optional]__ I think making use of JSX rather than the `React.createElement()` syntax makes your code more readable and easier to maintain. What do you think?

Reply:
I think is the same. At the end of the day both with JSX and React method will only need to change the options (type, etc) and the content (name).